packages/client/src/StreamVideoClient.ts

```diff
@@ -183,6 +184,11 @@ export class StreamVideoClient {
 
     this.eventHandlersToUnregister.push(
       this.on('call.created', (event) => {
+        if (this.shouldRejectCallWhenBusy()) {
+          this.logger('info', 'Call rejected because user is busy');
+          return;
+        }
```

Comment:
I'm not sure if we need this here. The backend will emit `call.created` when you are added as a member. It doesn't necessarily mean the call would be a `ringing`call.